xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -320,9 +326,11 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the maximum size of the header portion from the server response in bytes.</summary>
```

Comment:
```suggestion
        <summary>Gets or sets the maximum length, in kilobytes (1024 bytes), of the response headers. For example, if the value is 64, then 65536 bytes are allowed for the maximum response headers' length.</summary>
```

Reply:
The example portion should probably go in the remarks section instead of the summary.